app-connector-client/server/resources/api.yaml

```diff
@@ -293,6 +396,37 @@ paths:
 
 components:
   schemas:
+    Status:
+      type: "object"
+      properties:
+        success_count:

```

Comment:
Please use camelCase for the attribute names, also the variable names in the js files

Reply:
done